frontend/language/src/nb.json

```diff
@@ -159,7 +159,6 @@
   "code_list_editor.column_title_value": "Verdi",
   "code_list_editor.delete_code_list_item": "Slett alternativ {{number}}",
   "code_list_editor.description_item": "Beskrivelse for alternativ {{number}}",
-  "code_list_editor.empty": "Kodelisten er tom.",
```

Comment:
Are we sure we are going to remove this? 
The codelist is still technically empty until we have clicked the "Legg til alternativ" button.

Reply:
I agree. This is now restored.